.github/workflows/ruby.yml

```
@@ -0,0 +1,33 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Send lunch menu
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 10 * * 1-6'
```

```suggestion
    - cron:  '30 10 * * 1-5'
```
Or else it would also run on saturday.